### ISSUE TITLE
Computed properties

### ISSUE THREAD

**Issue by BCleverly** [ermouth/jQuery.my#30]
VueJS has a system called computed properties which can be used along side the data attributes setup. that can be manipulated before output.

Is this available within $.my? I'm doing some calculations where a number increments programmatically but the value doesn't update in the front end if it's done this way. I've tested this out with just changing a string programmatically and it suffers the same issue.

**Comment by ermouth** [comment]
Can you please clarify the question, without using `something has X, where X is here`?

**Comment by BCleverly** [comment]
In my form i need a progress tracker, I've got the total count sorted. it's the progression I'm struggling with.

so in data i have two properties called `total` and `progress`.

Every time someone answers a question I need `progress` to increment by one, if they delete and answer, it decrements by one. So `progress` needs to be calculated every time a binding is updated or a property in the dataset is updated. 

Here's a link to some functionality VueJs has [VueJs Computed Properties](https://vuejs.org/guide/computed.html) which may explain it better. 

I've just tried added a function to the data property to see if I can do it that way, but I can't seem to get it work dynamically at the moment.
